tests/js/router/index.js

```diff
@@ -0,0 +1,8 @@
+"use strict";
+require("./router-nesting-tests");
+require("./router-params-tests");
+require("./router-static-tests");
+require("./router-wildcard-tests");
+
+// TODO:  This test causes spurious "assertion outside test context" in other tests if it's run at the same time.  Discuss with Antranig.
+//require("./router-static-failure-tests");
```

Comment:
Nevermind, I just realized that you meant that I should use `fluid.test.runTests` in the individual files.